compiler/rustc_typeck/src/impl_wf_check/min_specialization.rs

```diff
@@ -145,9 +145,11 @@ fn check_always_applicable(
 /// types are expressed in terms of the generics of `impl1`.
 ///
 /// Example
-///
+/// 
+/// ```rust
 /// impl<A, B> Foo<A> for B { /* impl2 */ }
 /// impl<C> Foo<Vec<C>> for C { /* impl1 */ }
+/// ```
```

Comment:
This will fail because the types aren't defined. I'd suggest using `ignore` since you don't actually care what the traits themselves are.
```suggestion
/// ```ignore
/// impl<A, B> Foo<A> for B { /* impl2 */ }
/// impl<C> Foo<Vec<C>> for C { /* impl1 */ }
/// ```
```